openedx/features/genplus_features/genplus_learning/api/v1/serializers.py

```diff
@@ -20,31 +21,53 @@
 from openedx.features.genplus_features.genplus_badges.models import BoosterBadgeAward
 from openedx.features.genplus_features.genplus.api.v1.serializers import TeacherSerializer
 from openedx.features.genplus_features.common.utils import get_generic_serializer
-from openedx.features.genplus_features.genplus_assessments.utils import get_student_skills_assessment_completion
+from openedx.features.genplus_features.genplus_assessments.utils import (
+    get_student_program_skills_assessment,
+    get_student_unit_skills_assessment,
+)
+
+
+class BaseSerializer(serializers.ModelSerializer):
+    def get_fields(self):
```

Comment:
What's the purpose of the BaseSerializer, can we add doc string

Reply:
It's just for passing context into nested serializers